lighthouse-core/config/experimental-config.js

```diff
@@ -22,6 +23,15 @@ const config = {
       'full-page-screenshot',
     ],
   }],
+  categories: {
+    // @ts-ignore: `title` is required in CategoryJson. setting to the same value as the default
```

Comment:
i just copied this style from https://github.com/GoogleChrome/lighthouse/commit/c75f1ba97989af60713e8401c770e5e28b10313b#diff-bda47a785358888ca8941d29bebfbe98